src/EA.Weee.Requests/Organisations/DirectRegistrant/UpdateSubmissionPaymentDetailsRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace EA.Weee.Requests.Organisations.DirectRegistrant
+{
+    using CuttingEdge.Conditions;
+    using EA.Prsd.Core.Mediator;
+    using EA.Weee.Core.DirectRegistrant;
+    using System;
+
+    public class UpdateSubmissionPaymentDetailsRequest : IRequest<bool>
+    {
+        public Guid DirectRegistrantId { get; private set; }
+
+        public PaymentStatus PaymentStatus { get; private set; }
+
+        public Guid PaymentSessionId { get; private set; }
+
+        public bool IsFinalState { get; private set; }
```

Comment:
Should these not be nullable if they're not mandatory, and have no Condition.Requires()

Reply:
The conditions check that the Guids are not the default value. The enum won't be null and the isFinalState should always be a true or false.